Author: Otto Kekäläinen <otto@seravo.fi>
Debian-Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=540153
Subject: tests not getting started on kFreeBSD
 As per #670722 I found that the test socket created to test
 file name truncation was barfing even for the shortened form.
 This patch was adapted from the similar one in MySQL 5.6 package.
Forwarded: no
Last-Update: 2014-08-07
--- a/mysql-test/lib/My/Platform.pm
+++ b/mysql-test/lib/My/Platform.pm
@@ -111,6 +111,9 @@ sub check_socket_path_length {
   # See Bug #45771
   return 0 if ($^O eq 'aix');
 
+  # See Debian bug #670722 - failing on kFreeBSD even after setting short path
+  return 0 if length $path < 40;
+ 
   require IO::Socket::UNIX;
 
   my $truncated= undef;
